ReactAndroid/build.gradle

```diff
@@ -464,6 +464,20 @@ dependencies {
     androidTestImplementation("androidx.test:runner:${ANDROIDX_TEST_VERSION}")
     androidTestImplementation("androidx.test:rules:${ANDROIDX_TEST_VERSION}")
     androidTestImplementation("org.mockito:mockito-core:${MOCKITO_CORE_VERSION}")
+
+    implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
```

Comment:
Where is swipe refresh used?

Reply:
In React core itself. I wanted to have looked into this but haven't had a chance yet. Somehow the swipe layout no longer compiles now without explicitly depending on it here.